### PR TITLE
Cancel listing#14

### PR DESCRIPTION
**Pull request by damwhit**
user is able to destroy listings for which all tickets are active and user is able to inactivate listings for which some tickets are not active.

### CODE REVIEW

File: app/views/shared/_user_sidebar_nav.html.erb [hunk 1]
```diff
@@ -4,7 +4,7 @@
       <h>hi <%= current_user.first_name.downcase %></p>
     </li>
     <li class="divider"></li>
-    <li><%= link_to "dashboard", user_dashboard_path(current_user) %></li>
+    <li><%= link_to "dashboard", user_dashboard_path %></li>
     <li><%= link_to "order history", user_orders_path(current_user) %></li>
```
Comment (erinnachen) [inline app/views/shared/_user_sidebar_nav.html.erb]: looks like you don't need current_user for either user_orders_path or user_listings_path

File: app/views/users/_signup_form.html.erb [hunk 2]
```diff
@@ -15,7 +15,7 @@
     </div>
     <div class="col-xs-6">
       <div class="form-group text-center form-div">
-        <%= form_for(@user) do |f| %>
+        <%= form_for(@user, url: user_path(@user)) do |f| %>
```
Comment (erinnachen) [inline app/views/users/_signup_form.html.erb]: this also isn't going to need `@user` in the path

File: spec/features/user/user_can_remove_listing_spec.rb [hunk 3]
```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
+RSpec.feature "UserRemovesLIsting", type: :feature do
+  include SpecHelpers
```
Comment (erinnachen) [inline spec/features/user/user_can_remove_listing_spec.rb:3]: typo capital I

File: app/views/shared/_user_sidebar_nav.html.erb (hunk 1 above)
Comment (damwhit) [inline app/views/shared/_user_sidebar_nav.html.erb]: Good call.  Done!

File: app/views/users/_signup_form.html.erb (hunk 2 above)
Comment (damwhit) [inline app/views/users/_signup_form.html.erb]: Good eye!

**Comment by cheljoh** [comment]
Looking good, nice work!

**Pull request merged**